internal/api/handlers/schedulers_handler_test.go

```diff
@@ -1152,7 +1150,7 @@ func TestSwitchActiveVersion(t *testing.T) {
 		mux.ServeHTTP(rr, req)
 
 		require.Equal(t, 500, rr.Code)
-		require.Contains(t, rr.Body.String(), "failed to schedule operation: failed to schedule switch_active_version operation")
+		require.Contains(t, rr.Body.String(), "internal error")
```

Comment:
Shouldn't we return status code 404 here?

Reply:
BTW, this change should not to be implemented in this PR

Reply:
No, there's another test that returns 404 when the scheduler version is not found. This one simply reproduces an internal server error. See line 1126